src/Sulu/Bundle/Sales/OrderBundle/Order/OrderPdfManager.php

```diff
@@ -104,20 +131,54 @@ public function getPdfName($order)
      */
     public function createOrderConfirmation(ApiOrderInterface $apiOrder)
     {
+        return $this->createOrderPdfDynamic(
+            $apiOrder,
+            $this->templateHeaderPath,
+            $this->templateConfirmationPath,    // Since this is the action for confirmation pdfs.
+            $this->templateFooterPath
+        );
+    }
+
+    /**
+     * Function to create a pdf for a given order using given templates.
+     *
+     * @param ApiOrderInterface $apiOrder
+     * @param string $template
+     *
+     * @return file
+     */
+    public function createOrderPdfDynamic(
+        ApiOrderInterface $apiOrder,
+        $templateHeaderPath = null,
```

Comment:
new param to get templatePath of real base to extend, default to confirmationPath